package.json

```
@@ -56,8 +58,15 @@
     ]
   },
   "prettier": {
+    "importOrder": [
+      "^react$",
+      "<THIRD_PARTY_MODULES>",
+      "^[.]"
+    ],
+    "importOrderTypeScriptVersion": "5.0.0",
     "plugins": [
-      "prettier-plugin-tailwindcss"
+      "prettier-plugin-tailwindcss",
+      "@ianvs/prettier-plugin-sort-imports"
```

```suggestion
      "@ianvs/prettier-plugin-sort-imports",
      "prettier-plugin-tailwindcss"
```
_inconspicuous whistling_ 👀